### PR TITLE
Add a drone file to see if we can replacete the travis build in drone

### PR DESCRIPTION
**Pull request by easternbloc**

### CODE REVIEW

**Comment by easternbloc** [comment]
@bendiggle it lives!!!!

File: package.json [hunk 1]
```diff
@@ -12,14 +12,16 @@
...
+    "codecept": "codeceptjs run ./acceptance_tests --steps",
     "dev": "npm-run-all --parallel watch:app watch:scss watch:js watch:translations",
     "watch:app": "NODE_ENV=development nodemon .",
     "watch:scss": "nodemon -e scss -x 'npm run sass'",
     "watch:js": "nodemon --watch assets/js -e js -x 'npm run browserify'",
     "watch:translations": "nodemon --watch apps/**/translations/src -x 'npm run hof-transpile'",
     "test": "NODE_ENV=test mocha",
     "test:ci": "npm run lint && npm run style && npm run test",
-    "test:acceptance": "codeceptjs run ./acceptance_tests --steps",
+    "test:acceptance": "npm run codecept",
```
Comment (bendiggle) [inline package.json:23]: Do we really need two different npm scripts (codecept, test:acceptance) when the do exactly the same thing?

Comment (easternbloc) [inline package.json:23]: We'll likely use test:acceptance to start phantomjs also but I can't find a nice way to do that at the moment and have it work across test and ci (as phantomjs-prebuilt is not playing nicely with our docker image (not sure why atm))

Comment (easternbloc) [inline package.json:23]: so yeah it's good to have the abstraction there

Comment (bendiggle) [inline package.json:23]: Fair enough!

**Comment by bendiggle** [comment]
LGTM!

**Pull request merged**